providers/wpar.rb

```diff
@@ -17,7 +17,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 require 'chef/mixin/shell_out'
-require 'wpars'
+begin
+  require 'wpars'
+rescue LoadError # rubocop:disable Lint/HandleExceptions
+  # This space left intentionally blank.
```

Comment:
Do we want to rescue this with a chef_gem install like we do in aws and other cookbooks?